src/XamlX/Exceptions.cs

```diff
@@ -14,7 +14,7 @@ class XamlParseException : XmlException
         {
         }
 
-        public XamlParseException(string message, IXamlLineInfo lineInfo) : this(message, lineInfo.Line, lineInfo.Position)
+        public XamlParseException(string message, IXamlLineInfo lineInfo) : this(message, lineInfo?.Line ?? 0, lineInfo?.Position ?? 0)
```

Comment:
Not related to the PR specifically, but I found it rather distracting, when compiler fails with NRE instead of actual exception, when node might be invalid or null.